tests/test_policy_scorer.py

```
@@ -0,0 +1,295 @@
+    ratings = {d: 3 for d in (IMPORTANCE_DIMS + RESEARCH_VALUE_DIMS)}
+    return {
+        "claim_id": "c1", "argument_role": "core_claim",
+        "counterfactual_impact": "high", "ratings": ratings, "reason": "r",
+    }
+
+
+def test_refined_claim_from_dict_valid_and_invalid():
+    rc = RefinedClaim.from_dict(_valid_refined_dict())
+    assert rc.claim_id == "c1" and rc.entities == ["X"]
+    # ungültiger claim_type
+    bad = _valid_refined_dict(); bad["claim_type"] = "banana"
```

Ruff (0.15.20)

[error] Multiple statements on one line (semicolon)

(E702)